infra/repository/main.tf

```diff
@@ -0,0 +1,34 @@
+terraform {
+
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = ">= 3.100.0"
+    }
+
+    github = {
+      source  = "integrations/github"
+      version = "6.1.0"
+    }
+  }
+
+  backend "azurerm" {
+    resource_group_name  = "terraform-state-rg"
+    storage_account_name = "tfdevdx"
```

Comment:
```suggestion
    storage_account_name = "tfproddx"
```

Reply:
Remember to import this repository into the state
`terraform import github_repository.this dx`